webui/react/src/e2e/models/common/ant/Tabs.ts

```diff
@@ -16,26 +12,13 @@ export class Tabs extends NamedComponent {
   });
 
   /**
-   * Returns a representation of a tab item with the specified id. The
-   * component type specified will be retuned for interaction with the tab.
+   * Returns a Tab in the Tabs component
    * @param {string} id - the id of the menu item
-   * @param {new (args: BaseComponentArgs) => T} tabComponent - the type of the component that will be open after the tab is clicked.
    */
-  typedTab<T extends BaseComponent>(
```

Comment:
this was a clever solution to a pattern i hadn't quite implemented yet. the problem with this implementation is that these tabs are meant to only represent the tab in the tabbar, not the conent below in the tabcontent. the tabs can simply be referred to as basecomponents since they don't have any children.

in the implementation, the component which uses a pivot will also define it's content. we can simply define the content in the same model as the source file without needing to pass any types through.